package.json

```diff
@@ -43,7 +43,7 @@
     "lint:styles": "stylelint ./addon/*.css",
     "build": "npm-run-all build:*",
     "build:deps": "mkdir -p addon/build/ && cp node_modules/react/umd/react.production.min.js node_modules/react-dom/umd/react-dom.production.min.js node_modules/react-dom/umd/react-dom-server.browser.production.min.js addon/build/ && babel --retain-lines addon/popup.jsx > addon/build/popup.js && babel --retain-lines addon/emailTemplates.jsx > addon/build/emailTemplates.js",
-    "build:ga": "mkdir -p addon/build && cp $(node -e 'console.log(require.resolve(\"testpilot-ga\"))') addon/build/testpilot-ga.js",
+    "build:ga": "mkdir -p addon/build && cp node_modules/testpilot-ga/dist/index.js addon/build && mv addon/build/index.js addon/build/testpilot-ga.js",
```

Comment:
Isn't it possible to rename the file during the `cp` and save the intermediate `mv` step?

<kbd>cp ./node_modules/testpilot-ga/dist/index.js addon/build/testpilot-ga.js</kbd>